lib/cli.js

```diff
@@ -59,7 +59,7 @@ var yargs = require("yargs"),
 
     HOST_MESSAGE = "If no hosts are given, a screenshot from localhost is displayed.",
 
-    COMMANDS = ["search", "path", "ia", "group"],
+    COMMANDS = ["search", "path", "ia", "group", "build-group"],
```

Comment:
Hmm yeah, my initial thinking was to make it a single command as well but I having a single command do two different things felt a bit weird.

Thinking about it a bit more, I think you're right here. Ultimately the result is the same. I'll change it to a single command.